### PR TITLE
Add lippu.fi

### PR DESCRIPTION
**Pull request by slothtown**

### CODE REVIEW

File: img/l/lippu.fi.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...127.8 164.1v-80h14.4v7.3c1.8-2.8 3.9-5 6.5-6.5s5.3-2.3 8.2-2.3c6.6 0 11.8 2.7 15.7 8.1s5.8 12.8 5.8 22.2-1.9 16.8-5.8 22.2c-3.9 5.5-9.1 8.2-15.7 8.2-2.9 0-5.6-.7-8.2-2.2-2.6-1.5-4.7-3.7-6.5-6.5V164h-14.4v.1zm32.7-63.8c-2-3.3-4.7-4.9-8.3-4.9-1.9 0-3.8.6-5.7 1.9s-3.3 2.9-4.3 4.8v21.8c1 1.9 2.4 3.5 4.3 4.9 1.9 1.3 3.8 2 5.7 2 3.6 0 6.4-1.6 8.3-4.9 2-3.3 2.9-7.6 2.9-12.8 0-5.3-.9-9.5-2.9-12.8zM215.4 134.7c-1.7 2.5-4 4.6-7.1 6.2-3 1.7-6.4 2.5-10 2.5-5.2 0-9.1-1.4-11.6-4.1-2.6-2.7-3.8-6.6-3.8-11.5V84.1h14.4v38.3c0 3 .7 5.1 2 6.4 1.3 1.3 3.2 2 5.7 2 2.2 0 4.2-.7 6-2 1.8-1.4 3.3-3 4.4-5V84.1h14.4V142h-14.4v-7.3zM239.4 127.6c1.9-1.8 4.1-2.8 6.5-2.8 2.5 0 4.6.9 6.5 2.8 1.8 1.8 2.8 4 2.8 6.6 0 2.5-.9 4.6-2.7 6.5-1.8 1.8-4 2.8-6.5 2.8-2.6 0-4.8-.9-6.6-2.8-1.8-1.8-2.8-4-2.8-6.5s.9-4.7 2.8-6.6zM262.6 96.7h-7.4V84.1h7.4v-4.6c0-5.9 1.5-10.5 4.6-13.8 3.1-3.3 7.2-5 12.3-5 3.8 0 7 .7 9.6 2l-2.6 10.9c-.9-.7-2.1-1.1-3.6-1.1-1.8 0-3.2.6-4.2 1.8-1 1.2-1.6 2.9-1.6 5.1V84h9.5v12.6h-9.5V142h-14.5V96.7zm33.1-23.6c-1.7-1.7-2.6-3.7-2.6-6.1s.9-4.5 2.6-6.2 3.7-2.6 6.1-2.6 4.4.9 6.1 2.6c1.7 1.7 2.5 3.8 2.5 6.2s-.8 4.4-2.5 6.1-3.7 2.5-6.1 2.5c-2.4 0-4.4-.9-6.1-2.5zm-1.2 11h14.4V142h-14.4V84.1z" class="st1"/></svg>
```
Comment (hkamran80) [inline img/l/lippu.fi.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64.3 64.5"><path fill="#feca27" d="M24.4 40.1c6.7 6.7 6.9 17.2 6.9 24.4H33c0-7.3.2-17.8 6.9-24.5 6.7-6.7 17.2-6.9 24.4-6.9v-1.7c-7.3 0-17.7-.2-24.4-6.9C33.2 17.8 33 7.3 33 0h-1.7c0 7.3-.2 17.7-6.9 24.4-6.7 6.7-17.2 6.9-24.4 6.9V33c7.2.2 17.7.4 24.4 7.1z"/></svg>
```

Here's an SVG with just the star

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: img/l/lippu.fi.svg (hunk 1 above)
Comment (slothtown) [reply img/l/lippu.fi.svg]: It should also have the text, as it's always used in the logo as can be seen in lippu.fi's graphical charter: https://www.lippu.fi/obj/media/FI-eventim/pdf/lippufi-GraafisetOhjeet_08042022.pdf

Comment (hkamran80) [reply img/l/lippu.fi.svg]: Sure, but most sites on 2fa.directory have just the icon, not the full logo

Comment (Carlgo11) [reply img/l/lippu.fi.svg]: The favicon is just the star.

Comment (slothtown) [reply img/l/lippu.fi.svg]: Alright, didn't realize that

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**